playwright/src/main/java/com/deque/html/axecore/playwright/AxeBuilder.java

```diff
@@ -474,13 +472,38 @@ private Object finishRun(ArrayList<Object> partialResults) {
     Page blankPage = browser.newPage();
     blankPage.evaluate(getAxeScript() + getAxeConfigure(hasRunPartial));
 
+    String partialResString = "";
```

Comment:
nit - 

This function is already quite large, breaking it out into its own function would allow commenting on what and why it was needed and can reference [proposal](https://github.com/dequelabs/axe-api-team/pull/312)